docs/resources/Channel.md

```diff
@@ -465,13 +488,16 @@ Embed types are "loosely defined" and, for the most part, are not used by our cl
 | icon_url?       | string | url of footer icon (only supports http(s) and attachments) |
 | proxy_icon_url? | string | a proxied url of footer icon                               |
 
+> info
+> The `proxy_icon_url` attribute should not be specified when sending an embed. If provided, it will be ignored.
+
 ###### Embed Field Structure
 
-| Field   | Type    | Description                                     |
-| ------- | ------- | ----------------------------------------------- |
-| name    | string  | name of the field                               |
-| value   | string  | value of the field                              |
-| inline? | boolean | whether or not this field should display inline |
+| Field   | Type    | Description                                                                                                      |
+| ------- | ------- | ---------------------------------------------------------------------------------------------------------------- |
+| name    | string  | name of the field                                                                                                |
+| value   | string  | value of the field                                                                                                |
+| inline? | boolean | whether or not this field should display inline. The default is to render this inline if not specified. |
```

Comment:
This suggestion may need some better wording, so please take it with a grain of salt
```suggestion
| inline? | boolean | whether or not this field should display inline. By default, the field will not be rendered inline unless specified. |
```

![image](https://user-images.githubusercontent.com/17960496/82755552-e515d280-9ddc-11ea-98ad-3c6770196a4a.png)